### PR TITLE
MoveObject

### PR DESCRIPTION
**Pull request by jaudick**
[UNITY](https://github.com/NextCenturyCorporation/ai2thor/pull/201)
Please read the unity pr description too

### CODE REVIEW

File: integration_tests/data/101.move_object.scene.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+  "name": "Rotate",
```
Comment (kdrumm) [inline integration_tests/data/101.move_object.scene.json:2]: Thomas just caught me on this too, but we should change the name to something more relevant.

**Review by ThomasSchellenbergNextCentury — COMMENTED** [review]
I love all the integration tests! I'm going to hold off on approving for now until we finalize any changes in the corresponding Unity PR.

**Comment by jaudick** [comment]
@kdrumm @ThomasSchellenbergNextCentury  this PR is ready for review again

**Comment by kdrumm** [comment]
> @kdrumm @ThomasSchellenbergNextCentury this PR is ready for review again. The tests now start at 102 because of kyles lips pr

Whoops! I could have changed mine since this PR was out before mine!  Good catch though.

**Review by kdrumm — APPROVED** [review]

File: machine_common_sense/action.py [hunk 2]
```diff
@@ -512,6 +512,63 @@ class Action(Enum):
...
+        "0",
+        "Apply a movement of 0.1 units to a nearby object. " +
+        "(objectId=string, " +
+        "lateral=int(default:0), " +
+        "straight=int(default:1), " +
+        "objectImageCoordsX=float, " +
+        "objectImageCoordsY=float)"
+    )
+    """
+    Apply a movement 0.1 units to a nearby object.
```
Comment (ThomasSchellenbergNextCentury) [inline machine_common_sense/action.py:526]: 1. Missing an `of`
2. Someone had suggested to me recently that we should more clearly define our units moving forward. The "units" here are "meters", correct? You can say that.

File: machine_common_sense/action.py [hunk 3]
```diff
@@ -512,6 +512,63 @@ class Action(Enum):
...
+    lateral : int
+        The x axis direction of movement on the object.
+        Can be -1, 0, 1. If only lateral is given,
+        straight will default to 0
+        Default: 0
+    straight : int
+        The z axis direction of movement on the object.
+        Can be -1, 0, 1. If only straight is given,
+        lateral will default to 0
+        Default: 1
```
[the remark this answers is not in the document]
Comment (jaudick) [reply machine_common_sense/action.py:550]: Yes they are

File: machine_common_sense/action.py (hunk 2 above)
Comment (jaudick) [reply machine_common_sense/action.py:526]: Yes they are meters I will add that

**Review by ThomasSchellenbergNextCentury — APPROVED** [review]

**Pull request merged**